.env.example

```
@@ -0,0 +1,17 @@
+# Required
+GITHUB_CLIENT_ID=your_github_client_id
+GITHUB_CLIENT_SECRET=your_github_client_secret
+CALLBACK_URL=your_callback_url
+
+REFERRALS_DATA_DIR=/app/data
+REFERRALS_DB_PATH=/app/data/referrals.db
+
+DOMAIN=your domain
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)